packages/plugin-ext/src/plugin/comments.ts

```diff
@@ -491,6 +491,7 @@ function convertToModeComment(thread: ExtHostCommentThread, commentController: C
     }
 
     const iconPath = theiaComment.author && theiaComment.author.iconPath ? theiaComment.author.iconPath.toString() : undefined;
+    const _date = theiaComment.timeStamp ? theiaComment.timeStamp.toJSON() : undefined;
```

Comment:
```suggestion
    const date = theiaComment.timeStamp ? theiaComment.timeStamp.toISOString() : undefined;
```

I think a better way to serialize the date is using an ISO date. this can be done by using [Date#toISOString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString).
Then I'd change the comment in [plugin-api-rpc-model.ts](https://github.com/eclipsesource/theia/pull/58/files#diff-090c314b5db1fac40ea5a8a22114a5cde9ac27a5115e497471b4891d2397d370) to something like:
`/** Date serialized as ISO string */`

Regarding naming: I think using the underscore here is against Theia's coding guidelines for variable names. AFAIK this is only used for ununsed method parameters and properties that have `get` and/or `set` methods